### DISCUSSION TITLE
History of bathymetry datasets - why are they the way they are? How to move forward?

### DISCUSSION THREAD

**Discussion by MargaretSiple-NOAA** [afsc-gap-products/StationAllocationAIGOA#11] [General]
From Mark re: the updated bathymetry:

> First, there is no complete bathymetry of the GOA and there most likely won't be in the near future due to large unmapped or poorly mapped areas.
> Instead I would direct you toward our much better and more complete revised Aleutians data set as a base for stratifying a bottom trawl survey.
> http://projects.nprb.org/#metadata/a90807da-6f92-4a49-9e1d-ff589d2f6dbd/project
> This project was designed and completed for revising the Aleutians strata.
> I have already restratified the AI with new bathy contours but left alone the old longitudinal and latitudinal divisions, which also need to be reconsidered for validity.
> For example, I would recommend using our definition of pass locations as a substitute for the old latitudinal divisions.
> 
> Second, judging from the methods statement that Ned provided, there are a lot of misunderstandings about our published and unpublished GOA data sets.
> I haven't reviewed any EFH bathy compilations and, as a rule, only use original data sets rather than compilations created by others.
> 
> The eGOA data set, extending from Cape Ommaney to Dixon Entrance was never completed and published because those data were really crappy and did not depict major seafloor features in that area. I recommend not using this unpublished bathy raster.
> Since attempting to map this area, I have been trying to locate additional data sets that would allow completing this area but have not gotten very far.
> Much of the shelf is just mapped really poorly because offshore navigation was difficult a century ago (depths are presumably ok but locations are not).
> 
> The cGOA data set was completed with upsampled ROMs data only for use in NPRB's GOA-IERP.
> Those 3 km ROMs data points were upsampled to 100 m simply to produce a version of cGOA that conformed to the spatial extent of ROMs oceanographic modeling that was being used for GOA-IERP.
> (Originally GOA-IERP was only intended to cover the Kodiak/Kenai and Yakutat areas as contrasts in west versus east GOA shelf areas, but the modeling component of GOA-IERP pointed out that their modeled fish larvae would need to travel between these areas and therefore additional bathy coverage outside of Prince William Sound was requested).
> I tacked on the PWS to the Kodiak/Kenai and Yakutat areas and rounded it out with the upsampled ROMS to satisfy the oceanographic modelers.
> A lot of the ROMs bathy is in very poor shape, not reliable, and interpolating from 3 km to 100 m did not help but having wrong bathy data seems to work ok for the oceanographers as they compensate elsewhere in their modeling.
> 
> The version of cGOA people should be using was published as a tech memo and here:
> https://www.afsc.noaa.gov/RACE/groundfish/Bathymetry/CentralGOA_1.htm
> as it does not contain any ROMs data.
> This only contains real, observed depths.
> 
> The methods paragraph Ned provided does not describe using bathy coverage of the area from Unimak to IOFM that is only available in our original and updated Aleutians data sets:
> Original AI:
> http://web.archive.org/web/20170513133740/https://www.afsc.noaa.gov/RACE/groundfish/Bathymetry/Aleutians.htm
> Revised AI (also see above):
> http://projects.nprb.org/#metadata/a90807da-6f92-4a49-9e1d-ff589d2f6dbd/project
> The wGOA data set ends at the False Pass area and just includes a little bit from the AI compilation for overlap.
> 
> So to make a complete map of the GOA bottom trawl survey area, you would need:
> the SE part (Unimak to IOFM) of one of our AI rasters
> the wGOA
> the cGOA
> Cook Inlet
> and some sort of substitute for our eGOA (Ommaney to Dixon Entrance).
> That's why I would steer you to the AI instead.
> 
> Make sense?
> -Mark

**Comment by Ned-Laman-NOAA** [comment]
We will need to nail down (more clearly than I apparently did in the methods text I shared) exactly what and how we came up with updated GOA bathymetry so that we're prepared to support our stratum boundaries in the future. Between Jodi Pirtle, Megsie, and I we should have all of the pieces from the EFH project that we built the bathymetry for if I haven't shared that already.

**Comment by zoyafuso-NOAA** [comment]
For some reason, the bathy raster I was using was a little bit different than those in the RACE_EFH_Variables folder. That's why the maps I put in [this discussion thread](https://github.com/afsc-gap-products/StationAllocationAIGOA/discussions/16#discussioncomment-3900940) looked different (thanks @Ned-Laman-NOAA for pointing that out!). I made the novice mistake years ago of not documenting how I got this raster so I'm not even sure where I got the raster I'm using, and I take responsibility for that. Regardless, the bit with the edges of the domain missing is still an issue. 

To right this, I need to confirm which base aigoa bathy to use and which bathys to use to patch those missing edges.

aigoa_bathyp1c was one of the rasters shared with me early in the analysis (located in Y:\RACE_EFH_variables\Variables\ArcMap\AIGOAbathy_Combined\aigoa_bathp1c), and it covers the whole goa domain (black), just with those missing patches (in red):

![image](https://user-images.githubusercontent.com/60302777/196810709-2f4214be-0d15-40cc-bbfc-64c9cbb48ed4.png)

We could then patch those red areas on the edges by using rasters:

ai: Y:/RACE_EFH_variables/ZimmBathymetry/AI/ai_bathy/
wGOA: Y:/RACE_EFH_variables/ZimmBathymetry/wGOA/wgoa_bathy/
cGOA: Y:/RACE_EFH_variables/ZimmBathymetry/CGOA/cgoa_bathy/

Using those three rasters to inform those missing cells does increase coverage of the bathy:

![image](https://user-images.githubusercontent.com/60302777/196811371-3731aa36-35f1-4afb-92f1-c9cc456bb70d.png)

with smaller remaining patches in the SE, which we can figure out how to approximate how they should be defined. 

Does that work for now?

**Reply by Ned-Laman-NOAA** [reply]
I think this works for now. Clearly we need to expend some effort putting together detailed methods and sources around the compilation of this composite surface.

**Reply by MargaretSiple-NOAA** [reply]
This also looks good to me. And we do need documentation about all our bathymetry as we prepare to update it. Maybe Zimmerman can help w this too.

**Reply by Ned-Laman-NOAA** [reply]
Documenting a conversation Zack and I had last night around this.
1 - I've recommended and Zack agreed that we'll create polygons based on our updated bathymetry and reflecting the coverage shown here and share those with Mark for discussion
2 - We'll also attempt to translate trawlable/untrawlable designations from the old grid/station milieu to the new and communicate those to Mark at the same time...Mark had concerns about how changing T/UT designations might impact the SIRRCA project focused on UT areas
3 - Mark's ability and available time to provide an ArcGIS version of the chart we'll need based on this new information will be a significant hurdle for us to get over on the road to implementing the restrat for FY23.
4 - We are hoping to get Mark's help/input on what to do with the voids in the present coverage and on documenting what he can contribute to the compilation of bathymetry that this surface represents.
5 - The other hurdle yet to be crossed is whether or not the SSMA folks will be ready to accept a different set of final tables than they have in the past. I finally have it clear in my head what they'll be getting and what they won't so I'll be sending that email out today. They are also requesting a focused meeting with us to provide a more detailed overview of the restratification, the why of it, and answer any questions that arise. I suspect that they'll be looking at a January time frame for a meeting like that, but we'll have to have decided before then if we're go on FY23 restrat or not.

Zack, If you recall any more or something different, please drop it in.
